app/build.gradle

```diff
@@ -48,6 +49,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.appcompat:appcompat:1.5.1'
```

Comment:
I'd keep appcompat here, so we have control about the version.

Reply:
Keep appcompat comes with a bug where you need to declare 'androidx.lifecycle..' https://stackoverflow.com/questions/73406969/duplicate-class-androidx-lifecycle-viewmodellazy-found-in-modules-lifecycle-view